kotlinx-coroutines-core/common/src/Builders.common.kt

```diff
@@ -161,11 +162,14 @@ public suspend fun <T> withContext(
             val coroutine = UndispatchedCoroutine(newContext, uCont)
             // There are changes in the context, so this thread needs to be updated
             withCoroutineContext(coroutine.context, null) {
+                @Suppress("LEAKED_IN_PLACE_LAMBDA") // Contract is preserved, invoked immediately or throws
                 return@sc coroutine.startUndispatchedOrReturn(coroutine, block)
             }
         }
         // SLOW PATH -- use new dispatcher
         val coroutine = DispatchedCoroutine(newContext, uCont)
+        // Contract is preserved, invoked immediately or throws
+        @Suppress("LEAKED_IN_PLACE_LAMBDA")
```

Comment:
I'd prefer that the comment is to the right of `Suppress`: typically, the comment above a line with a suppression refers to the line itself.